CHT/Temperature.C

```diff
@@ -21,10 +21,27 @@ void preciceAdapter::CHT::Temperature::write(double* buffer, bool meshConnectivi
 
     if (this->locationType_ == LocationType::volume)
     {
-        forAll(T_->ref(), i)
+        if(cellSetNames_.empty())
         {
-            buffer[bufferIndex++] = T_->ref()[i];
+            forAll(T_->ref(), i)
+            {
+                buffer[bufferIndex++] = T_->ref()[i];
+            }
+        }
+        else
+        {
+            for (uint j = 0; j < cellSetNames_.size(); j++) {
```

Comment:
So, this should also work for multiple cell sets, without the one affecting the other, correct? It would be nice to showcase at least two cell sets in the respective tutorial.